### PR TITLE
chore: remove minimum version pins for pins older than 6 mo

### PR DESCRIPTION
**Pull request by Coniferish**
Remove a number of pins in `requirements/deps/constraints.txt` and `make pip-compile`

### CODE REVIEW

**Comment by Coniferish** [comment]
@MthwRobinson or anyone else. Do you know how this passed the CI checks without having the changelog or __version__ updated?

**Comment by MthwRobinson** [comment]
@Coniferish - The changelog check only runs if you change the package code under `unstructured`, so that's why CI skipped it for this PR

https://github.com/Unstructured-IO/unstructured/blob/f440eb476cf75d6109e8a3719cadf893529dcef8/.github/workflows/ci.yml#L529-L532

File: requirements/deps/constraints.txt [hunk 1]
```diff
@@ -5,28 +5,14 @@
 ####################################################################################################
 # consistency with local-inference-pin
 protobuf<4.24
-# NOTE(robinson) - Required pins for security scans
```
Comment (MthwRobinson) [inline requirements/deps/constraints.txt:8]: 🎉

**Review by MthwRobinson — APPROVED** [review]
Nice cleanup!

**Pull request merged**